README.rst

```diff
@@ -310,6 +310,60 @@ __ https://docs.scale.com/reference/set-metadata
     new_metadata = {'myKey': 'myValue'}
     task.set_metadata(new_metadata)
 
+Set A Task's Tags
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set a given task's ``tags``. Check out `Scale's API documentation`__ for more information.
```

Comment:
```suggestion
Set a given task's ``tags``. This will replace all existing tags on a task. Check out `Scale's API documentation`__ for more information.
```